app/assets/javascripts/helpers/select_handler.js

```diff
@@ -100,10 +121,24 @@
 
     _onChangeSelectValue: function() {
       this.$select.change(function(e) {
-        this._setHaveValue();
-        this._setSpeaker(this.$el.find("option:selected").text());
-        this._closeProcess();
-        this._runCallback();
+        var value = this.$select.val();
+        if(value == "") {
```

Comment:
same in other bits of this file. Just trying to understand if it's an issue or not! =)